create.go

```diff
@@ -0,0 +1,322 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/codegangsta/cli"
+	"github.com/hyperhq/runv/containerd/api/grpc/types"
+	"github.com/kardianos/osext"
+	"github.com/opencontainers/runtime-spec/specs-go"
+	netcontext "golang.org/x/net/context"
+)
+
+func firstExistingFile(candidates []string) string {
+	for _, file := range candidates {
+		if _, err := os.Stat(file); err == nil {
+			return file
+		}
+	}
+	return ""
+}
+
+func getDefaultBundlePath() string {
+	cwd, err := os.Getwd()
+	if err != nil {
+		return ""
+	}
+	return cwd
+}
+
+var createCommand = cli.Command{
+	Name:  "create",
+	Usage: "create a container",
+	ArgsUsage: `<container-id>
+
+Where "<container-id>" is your name for the instance of the container that you
+are creating. The name you provide for the container instance must be unique on
+your host.`,
+	Description: `The create command creates an instance of a container for a bundle. The bundle
+is a directory with a specification file named "` + specConfig + `" and a root
+filesystem.
+
+The specification file includes an args parameter. The args parameter is used
+to specify command(s) that get run when the container is started. To change the
+command(s) that get executed on start, edit the args parameter of the spec. See
+"runv spec --help" for more explanation.`,
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  "bundle, b",
+			Value: getDefaultBundlePath(),
+			Usage: "path to the root of the bundle directory, defaults to the current directory",
+		},
+		cli.StringFlag{
+			Name:  "console",
+			Usage: "specify the pty slave path for use with the container",
+		},
+		cli.StringFlag{
+			Name:  "pid-file",
+			Usage: "specify the file to write the process id to",
+		},
+		cli.BoolFlag{
+			Name:  "no-pivot",
+			Usage: "[ignore on runv] do not use pivot root to jail process inside rootfs.  This should be used whenever the rootfs is on top of a ramdisk",
+		},
+	},
+	Action: func(context *cli.Context) {
+		runContainer(context, true)
+	},
+}
+
+func runContainer(context *cli.Context, createOnly bool) {
+	root := context.GlobalString("root")
+	bundle := context.String("bundle")
+	container := context.Args().First()
+	ocffile := filepath.Join(bundle, specConfig)
+	if context.String("console") != "" {
+		fmt.Fprintf(os.Stderr, "--console is unsupported on runv\n")
```

Comment:
As `--console` is unsupported flag and latest `runc` has replaced `--console` with `--console-socket` flag, I suggest we remove `--console` and add `--console-socket`, and here it will become:

```
fmt.Fprintf(os.Stderr, "--console-socket is unsupported on runv\n")
```